unit_test/validate_schema_test.py

```diff
@@ -12,15 +12,22 @@
 DIR = Path(__file__).parent.resolve()
 
 
-def test_validate_default_schema():
+@pytest.fixture(scope="session")
+def validator() -> validate_pyproject.api.Validator:
+    """
+    Reuse the validator for all tests, to keep unit tests fast.
+    """
+    return validate_pyproject.api.Validator()
```

Comment:
Historical note: this used to accidentally mutate itself, so a year or two ago this might not have worked. :) (it was injecting defaults, so it probably would still work and I just was really unlucky that I was affected by defaults being injected elsewhere)

Reply:
Ah, good to know :)